scripts/cmake/findPETSC/FindPETSc.cmake

```diff
@@ -109,7 +109,13 @@ find_package_multipass (PETSc petsc_config_current
 
 # Determine whether the PETSc layout is old-style (through 2.3.3) or
 # new-style (>= 3.0.0)
-if (EXISTS "${PETSC_DIR}/${PETSC_ARCH}/include/petscconf.h")   # > 2.3.3
+
```

Comment:
Did you update FindPETSc from https://github.com/jedbrown/cmake-modules/blob/master/FindPETSc.cmake?